addons/web/static/src/views/list/list_renderer.js

```diff
@@ -1249,6 +1243,7 @@ export class ListRenderer extends Component {
                             futureRecord.switchMode("edit");
                         }
                     } else {
+                        list.unselectRecord(true);
```

Comment:
I tried this. But I didn't do the "skipTogglerTabbing" props. With this change, the test I introduced is not needed, right?

I guess it's a more superior solution because it will also remove the focus on the list field.